src/code/z_olib.c

```diff
@@ -100,37 +100,37 @@ Vec3f* OLib_VecSphToVec3f(Vec3f* dest, VecSph* sph) {
 }
 
 /**
- * Takes the geographic point `sph` and converts it into a x,y,z position
+ * Takes the geographic point `geo` and converts it into a x,y,z position
```

Comment:
Spherical Coordinates:
https://en.wikipedia.org/wiki/Spherical_coordinate_system
Pitch is defined from the top (north), what would be the y-axis in OoT is 0 pitch

Geographic Coordinates:
https://en.wikipedia.org/wiki/Geographic_coordinate_system
Pitch is defined from looking forward (horizon), what would be the xz-plan in OoT is 0 pitch, where positive pitch is upwards.

Reply:
> The geographic coordinate system (GCS) is a [spherical](https://en.wikipedia.org/wiki/Spherical_coordinate_system) or [ellipsoidal coordinate system](https://en.wikipedia.org/wiki/Ellipsoidal_coordinates_(geodesy)) for measuring and communicating [positions](https://en.wikipedia.org/wiki/Position_(geometry)) directly on the [Earth](https://en.wikipedia.org/wiki/Earth) as [latitude](https://en.wikipedia.org/wiki/Latitude) and [longitude](https://en.wikipedia.org/wiki/Longitude).

But this isn't about Earth (note the use of "The"), this is just a spherical coordinate system that uses the latitude instead of the more common colatitude. Wikipedia also highlights that GCS uses degrees instead of radians, which makes it an even worse analogy. See also
> To a first approximation, the [geographic coordinate system](https://en.wikipedia.org/wiki/Geographic_coordinate_system) uses elevation angle ([latitude](https://en.wikipedia.org/wiki/Latitude)) in degrees north of the [equator](https://en.wikipedia.org/wiki/Equator) plane, [...] 

(https://en.wikipedia.org/wiki/Spherical_coordinate_system#In_geography).

It seems like there is no general term for this, but "geographic" isn't the right one.

Reply:
There still needs to be a distinction between these two coordinate systems. Using spherical for both conventions, as was done previously, led to a lot of confusion working through camera calculations, so two distinct words are needed.

Internally in OLib, the term `geo`/`geographic` was already in use, so that's what I stuck with. It seems to be the closest coordinate system to a first order approximation, although if you can find a better word, then we can switch to that!